packages/common/src/services/wallet-client/WalletClient.ts

```diff
@@ -253,13 +255,10 @@ export class WalletClient {
         }
         console.error(
           `Error sending sol wrapped audio amount ${amount.toString()} to ${address.toString()}` +
-            `with error ${error.toString()} and errorCode: ${errorCode}`
-        )
-        throw new Error(
-          `Error: ${error.toString()}, with code ${errorCode?.toString()}`
+            `with error ${error.toString()}`
         )
+        throw new Error(`Error: ${error.toString()}`)
```

Comment:
nit: 
* I don't think the `Error: ` prefix adds anything here.
* Are we converting an error to a string and then wrapping it in an error again? We should just throw the original error here
* If we're already going to console.error below, I don't think we need to do it again here.

I guess just generally suggesting a less complicated error pattern that will play well with any sentry reporting that's happening above this.

Reply:
updated, check it out